server/index.js

```diff
@@ -2,14 +2,18 @@
 
 const app = require('./app');
 const log = require('./logger');
-const conf = require('./config')();
 
-require('./mocks');
-require('./fallback');
-require('./dead-end');
+module.exports = options => {
+  const config = require('./config')(options);
```

Comment:
initializes the config module just once and send it as a parameter to the other modules that needs it 👇